src/main/java/com/alkemy/ong/security/SecurityConfig.java

```diff
@@ -0,0 +1,41 @@
+package com.alkemy.ong.security;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+@EnableWebSecurity
+@EnableGlobalMethodSecurity(prePostEnabled = true)
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Autowired
+    private UserDetailsServiceImpl userDetailsService;
+
+    // AUTHENTICATION
+    @Override
+    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+        auth.authenticationProvider(authProvider());
+    }
+
```

Comment:
en esta clase, se puede agregar un metodo para poner de forma centralizada las rutas a securizar (en lugar de en los controllers). Te paso un ej:
@Override
 protected void configure(HttpSecurity http) throws Exception {
        .antMatchers(HttpMethod.GET, "/lucas-paoliello/users").hasAuthority("Super ADMIN 51")
                .anyRequest().authenticated()

Reply:
.and().formLogin().permitAll(); Agregue eso para que funcione en el navegador... En postman no funciona, me da 403 todo el tiempo. No se como hacerlo...